### PR TITLE
vegeta 5.6.3

### PR DESCRIPTION
**Pull request by tsenart**

### CODE REVIEW

File: Library/Formula/vegeta.rb [hunk 1]
```diff
@@ -1,9 +1,10 @@
 require "language/go"
 
 class Vegeta < Formula
+  @@version = "v5.6.3"
   homepage "https://github.com/tsenart/vegeta"
-  url "https://github.com/tsenart/vegeta/archive/v5.6.2.tar.gz"
-  sha1 "76c4b44ebf860343a638a71b2e574a8358b08fd8"
+  url "https://github.com/tsenart/vegeta/archive/#{@@version}.tar.gz"
```
Comment (mikemcquaid) [inline Library/Formula/vegeta.rb]: Don't inline this, thanks.

File: Library/Formula/vegeta.rb [hunk 2]
```diff
@@ -1,9 +1,10 @@
 require "language/go"
 
 class Vegeta < Formula
+  @@version = "v5.6.3"
```
Comment (mikemcquaid) [inline Library/Formula/vegeta.rb]: Remove this.

**Comment by tsenart** [comment]
Done.

**Comment by tdsmith** [comment]
Thanks for the pull request!

**Pull request closed without merging**